[32mINFO    [0m root:test_calculator.py:141 参数： 文, 9.3  预期异常： TypeError
[32mINFO    [0m root:test_calculator.py:145 实际异常： TypeError